### DISCUSSION TITLE
Steps?

### DISCUSSION THREAD

**Discussion by Shaimaa177** [galaxyproject/training-material#2511] [Q&A] [answered]
![received_302492498011158.jpeg](https://user-images.githubusercontent.com/81324022/115487597-519d8980-a259-11eb-90af-fc65e6c526d6.jpeg)

Which steps should I follow on Galaxy to answer this question?

**Comment by hexylena** [comment, marked as answer]
Hi @Shaimaa177, I'm not sure this is part of the galaxy training materials, please check with your tutorial provider. And in the future please consider asking on the Galaxy Help forum https://help.galaxyproject.org where more people can provide help.
